Core/src/main/java/com/plotsquared/core/configuration/Settings.java

```diff
@@ -666,6 +666,8 @@ public static final class Paper_Components {
         public static boolean TILE_ENTITY_CHECK = true;
         @Comment("Use Paper's async tab completion")
         public static boolean ASYNC_TAB_COMPLETION;
+        @Comment("Allow beacon effects originating from other plots affecting the own plot")
+        public static boolean BEACON_EFFECTS = false;
```

Comment:
This configuration option is too ambiguous, how about `cross plot beacon effects`?

Reply:
You mean ``Allow cross plot beacon effects``? Okay, should I change it in the docs as well?

Reply:
Please do that, thanks.